### PR TITLE
Home Page Copy Update & Main.css

### PR DESCRIPTION
**Pull request by OJ423**
Updated index.html page copy
Updated H1, .hero-banner__heading, & .block__heading styles in main.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,735 +1,741 @@
...
+		<main>
+
+			<section class="hero-banner">
+				<div class="wrapper hero-banner__wrapper">
+					<div class="hero-banner__col hero-banner__col--graphic">
+						<img src="img/placeholders/placeholder-banner.svg" aria-hidden=”true” class="hero-banner__image" alt="">
+					</div>
+					<div class="hero-banner__col hero-banner__col--text">
+						<h1 class="h1 hero-banner__heading">Build Collaborative Applications</h1>
+						<p class="hero-banner__description">TerminusDB is a toolkit for building collaborative applications. If your application or data product needs approval workflows, you can use TerminusDB to implement. TerminusDB enables Git-like collaborative workflows in your application.</p>
```
Comment (spl) [inline index.html:174]: > you can use TerminusDB to implement

Suggested alternatives:
* you can use TerminusDB to implement **them**
* you can implement **them** with TerminusDB

File: index.html [hunk 2]
```diff
@@ -1,735 +1,741 @@
...
+
+
+			<section class="page-strip">
+				<div class="primary-grid">
+					<a href="/blog/critical-asset-management-system/" target="_blank" rel="noreferrer" class="block">
+						<div class="block__content">
+							<i class="icon-decentralized block__icon"></i>
+							<div class="block__inner">
+								<h3 class="h3 block__heading">Build Collaborative Apps Fast</h3>
+								<<p class="block__description">Develop fast - an MVP in a single sprint -  and give your users the safety of ‘undo’.  Implement flexible and infinitely extendable schema to blueprint applications. No more f*****g around, do what you love with TerminusX.</p>
```
Comment (spl) [inline index.html:273]: There are 4 `<<` throughout this document. You can see the effect of one below on the page rendering.

extra-lt

 Can you change each to `<`?

**Review by spl — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (OJ423) [reply index.html:174]: Thanks for these - I wasn't comfortable with this sentence. Will make the change.

File: index.html (hunk 2 above)
Comment (OJ423) [reply index.html:273]: Yes, I noticed these and thought they were odd. I checked the original from Kooba and they were there so left them in. 
I wasn't sure if they were actually meant to be there. I'll remove as they look better without.

Comment (OJ423) [reply index.html]: Updated these now.

File: index.html (hunk 1 above)
Comment (OJ423) [reply index.html]: Reworded this to combine the last two sentences.

**Review by spl — APPROVED** [review]

**Pull request merged**